upup/pkg/fi/cloudup/openstacktasks/subnet.go

```diff
@@ -123,7 +123,7 @@ func (_ *Subnet) CheckChanges(a, e, changes *Subnet) error {
 		if changes.DNSServers != nil {
 			return fi.CannotChangeField("DNSServers")
 		}
-		if changes.Network != nil {
+		if fi.StringValue(a.Network.ID) != fi.StringValue(e.Network.ID) {
```

Comment:
Not sure if any member of network is allowed to change.  Not sure how to trigger than however as its based on the cluster name.  This might be fine?